Gems/Terrain/Code/Source/TerrainRenderer/Vector2i.cpp

```diff
@@ -38,4 +55,42 @@ namespace Terrain
     {
         return {-m_x, -m_y};
     }
+    
+    auto Vector2i::operator*(const Vector2i& rhs) const -> Vector2i
```

Comment:
Sometimes I'll use this form if the return type is some type defined inside the class because it cuts down on the number of characters. That's not the case here, I'm guessing it made sense in an earlier revision. I'll switch it back to standard form.